product_pricelist_alternative/views/product_pricelist_view.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_pricelist_view" model="ir.ui.view">
+        <field name="name">product.pricelist.form</field>
+        <field name="model">product.pricelist</field>
+        <field name="inherit_id" ref="product.product_pricelist_view" />
+        <field name="arch" type="xml">
+            <widget name="web_ribbon" position="before">
+                <div class="oe_button_box" name="button_box">
+                    <button
+                        class="oe_stat_button"
+                        type="object"
+                        name="action_view_is_alternative_to_pricelist"
+                        icon="fa-list"
+                    >
+                        <field
+                            string="Alternative to Pricelist"
+                            name="is_alternative_to_pricelist_count"
+                            widget="statinfo"
+                        />
+                    </button>
+                </div>
+            </widget>
+
+            <group name="pricelist_settings" position="after">
```

Comment:
I would add a specific tab instead and list the pricelists instead of using m2m tags

Reply:
Not sure that it is a better option. New tab implies one more click to the user to check alternative pricelists.

Reply:
I imagine that price list managers access this forms every day multiple times. And one more click makes the difference. I ask the BA...
If not then it can go to a new tab indeed